eclair-cli/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2019 ACINQ SAS
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>fr.acinq.eclair</groupId>
+    <artifactId>eclair-cli</artifactId>
+    <version>1.0.0</version>
```

Comment:
Shouldn't we align version numbers with `eclair`?
Can't we inherit that from the parent?

Reply:
The project is not a child module of the `eclair` project for technical reasons:

> has no impact on the current build [...] also prevent packaging the scala-lib dependency (defined in the root pom) into the assembly JAR.

But this can probably be solved by tweaking the assembly plugin configuration.

Reply:
> The project is not a child module of the eclair project

Yeah that makes sense, but I don't know how we can keep their versions aligned...
Maybe we don't really care though :)